static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -354,7 +354,7 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
   }
 
   // No platform or no docs (new nor old)
-  if (!currentPlatform || (!newDocs && !hasOnboardingContents)) {
+  if (!currentPlatform || ((!newDocs || !dsn) && !hasOnboardingContents)) {
```

Comment:
i don't think we should add the `!dsn` check here because we still want to fallback to the old onboarding contents even if there isn't a `dsn`; however this logic means the entire block won't render if the `dsn` isn't available
```suggestion
  if (!currentPlatform || (!newDocs && !hasOnboardingContents)) {
```

Reply:
I updated the code because I was running into a strange state:

![image](https://github.com/getsentry/sentry/assets/29228205/83efe05b-37c1-44c0-a17f-9a65b12948e2)

However, I just realized this issue arises because the old docs don't work for the de region.

![image](https://github.com/getsentry/sentry/assets/29228205/1660cbc5-b7d4-4734-91ec-f99c192a987c)

should we look into this?

Reply:
hmm that's odd... maybe let's leave as-is for now then 🤔

cc @ArthurKnaus on this issue, maybe he knows more about it?

Reply:
looks like some old docs _do_ work, like react & svelte for example, but others don't

Reply:
I am not aware of any issue with the de region. Are we confident that it is de only? Or were the docs deleted / moved in the sentry-docs repo?

Reply:
ohh i think the old docs may have been deleted. old docs were the `.md` files we were using previously. honestly these docs are super old and we've migrated all of replay to the new docs structure so i'd be okay with cleaning up all references to the old docs! happy to make a PR to clean up the replays onboarding

Reply:
ah actually -- i think what @priscilawebdev is running into may be the fact that for some platforms, we didn't have old docs written for replay, so they didn't exist in the first place

Reply:
https://github.com/getsentry/sentry/pull/74025 here's the PR for cleaning up the references to the old onboarding docs. can wait to merge after this one gets merged in, because there'll be some merge conflicts